exporter/awsemfexporter/emf_exporter.go

```diff
@@ -197,9 +189,38 @@ func (emf *emfExporter) listPushers() []cwlogs.Pusher {
 }
 
 func (emf *emfExporter) start(_ context.Context, host component.Host) error {
+	// Create AWS session here
+	awsConfig, session, err := awsutil.GetAWSConfigSession(emf.config.logger, &awsutil.Conn{}, &emf.config.AWSSessionSettings)
+	if err != nil {
+		return err
+	}
+
+	// create CWLogs client with aws session config
+	svcStructuredLog := cwlogs.NewClient(emf.config.logger,
+		awsConfig,
+		emf.set.BuildInfo,
+		emf.config.LogGroupName,
+		emf.config.LogRetention,
+		emf.config.Tags,
+		session,
+		cwlogs.WithEnabledContainerInsights(emf.config.IsEnhancedContainerInsights()),
+		cwlogs.WithEnabledAppSignals(emf.config.IsAppSignalsEnabled()),
+	)
+
+	// Assign to the struct
+	emf.svcStructuredLog = svcStructuredLog
+
+	if emf.config.IsAppSignalsEnabled() {
+		userAgent := appsignals.NewUserAgent()
```

Comment:
A userAgent would have already been created in the constructor. Should we re-use here in the next line that instead of creating a new one?